### PR TITLE
Tt/sqlite

### PR DESCRIPTION
**Pull request by timothytong**
ISSUE: https://github.com/PeterChu2/WATPlanner/issues/12

### CODE REVIEW

File: app/src/main/java/com/example/peterchu/watplanner/Models/Course.java [hunk 1]
```diff
@@ -18,6 +19,7 @@
     private String title;
 
     public Course(String subject, String number, String credits, String title) {
+        this.id = -1;
```
Comment (PeterChu2) [inline app/src/main/java/com/example/peterchu/watplanner/Models/Course.java:22]: why do we do this? Can't we just pass in the ID in the constructor?

File: app/src/main/java/com/example/peterchu/watplanner/Database/DBHandlerCallback.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.example.peterchu.watplanner.Database;
+
+/**
+ * Created by Timothy Tong on 6/11/17.
+ */
+
+public interface DBHandlerCallback {
+    void callback(DatabaseHandler dbHandler);
```
Comment (PeterChu2) [inline app/src/main/java/com/example/peterchu/watplanner/Database/DBHandlerCallback.java]: can be more descriptive method name, e.g. onFinishTransaction?

**Review by PeterChu2 — APPROVED** [review]
small nits, but LGTM! :shipit:

File: app/src/main/java/com/example/peterchu/watplanner/Models/Course.java (hunk 1 above)
Comment (timothytong) [reply app/src/main/java/com/example/peterchu/watplanner/Models/Course.java:22]: Retrofit automatically converts the Course objects in the JSON directly to Course objects in Java. The arguments have to match (and id of course woudln't exist in the JSON returned by the API)

**Pull request merged**